src/pkg/portforwarder/portforwarder.go

```diff
@@ -76,10 +76,7 @@ func (p *PortForwarder) Start(ctx context.Context) (localPort int, err error) {
 			panic(err)
 		}
 	}()
-	select {
-	case <-readyChan:
-		break
-	}
+	<-readyChan
```

Comment:
pkg/portforwarder/portforwarder.go:79:2: S1000: should use a simple channel send/receive instead of `select` with a single case (gosimple)